app/experimenter/nimbus-ui/src/components/Summary/TableBranches/index.tsx

```diff
@@ -69,7 +69,16 @@ const TableBranch = ({
   branch: Branch;
 }) => {
   return (
-    <Table bordered data-testid="table-branch" className="mb-4" id={slug}>
+    <Table
+      bordered
+      data-testid="table-branch"
+      className="mb-4 table-fixed"
+      id={slug}
+    >
+      <colgroup>
+        <col className="w-33" />
+        <col />
+      </colgroup>
```

Comment:
Hey @anthonytran7757 can you elaborate on what this empty colgroup is doing? Is this a hacky way of fixing the problem, or could there be a better way?

Reply:
Hey @jodyheavener I've tried some other things, but this was the only one that worked for me aside from setting a colSpan={1} or colSpan={2} for each of the cells to maintain the 33% to 66% ratio between the columns. Could I get some guidance to find a less hacky solution?

Reply:
@anthonytran7757 I did some more research into this and found it to be an OK approach, and also couldn't find a better way of doing this without removing the thead. If you want to squash and reword your commit messages I can merge 👍